components/dialog/overview.md

```diff
@@ -53,6 +53,12 @@ The Dialog component and its [predefined options]({%slug dialog-predefined%}) ai
 
 Predefined Dialogs are styled substitutes to the standard browser dialogs - confirm, alert and prompt. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
 
+## Integration With Other Components
```

Comment:
As the added content seems implementation-specific, I recommend establishing a new page instead of adding it to the Overview.